warehouse/organizations/models.py

```diff
@@ -189,13 +217,19 @@ class Organization(HasEvents, db.Model):
         nullable=True,
         onupdate=func.now(),
     )
+    customer_id = Column(
+        Text, nullable=True, unique=True
+    )  # generated by Payment Service Provider
 
     users = orm.relationship(
         User, secondary=OrganizationRole.__table__, backref="organizations", viewonly=True  # type: ignore # noqa
     )
     projects = orm.relationship(
         "Project", secondary=OrganizationProject.__table__, back_populates="organization", viewonly=True  # type: ignore # noqa
     )
+    subscriptions = orm.relationship(
```

Comment:
Maybe not worth it in initial implementation since we _do_ plan to go with stripe, but should this be through another model? Like `StripeSubscriptions`?

Reply:
I misread this one! I think the relation through `OrganizationSubscription` is good!